README.rst

```diff
@@ -139,15 +139,15 @@ The following code implements a simple dual moving average algorithm.
                long_mavg=long_mavg)
 
 
-You can then run this algorithm using the Zipline CLI. From the command
-line, run:
+You can then run this algorithm using the Zipline CLI; you'll need a `Quandl <https://www.quandl.com/>`__ API key for default data bundle.
+Once you have you key, rom the command line, run:
```

Comment:
"Once you have you key, rom the command line, run:" -> "Once you have your key, run the following from the command line:"